TaylorFramework/Modules/finder/ErrorPrinter.swift

```diff
@@ -23,10 +23,10 @@ struct ErrorPrinter {
     }
     
     func printWrongFilePath(wrongPath: FilePath) {
-        printer.printError("File path not exist or type is wrong \(wrongPath)")
+        printer.printError("File path doesn't exist or type is wrong \(wrongPath)")
     }
     
-    func printFileManagerError(path: FilePath) {
-        printer.printError("File manager has not found subpath from \(path)")
+    func printSubpathsError(directoryPath path: FilePath) {
```

Comment:
Wouldn't it be better to have an enum of errors rather than creating a new function for each case? It's not scalable